ui/src/views/Clients/ViewClient/index.jsx

```diff
@@ -225,8 +225,10 @@ export default class ViewClient extends Component {
 
       // CLI login
       if (callbackUrl) {
+        const encodedClientId = encodeURIComponent(clientId);
+        const encodedAccessToken = encodeURIComponent(result.data.createClient.accessToken);
```

Comment:
I think this is redundant, as generated access token are already URL-safe as they are generated using https://github.com/taskcluster/slugid/ library